Example/Sources/ConversationViewController.swift

```diff
@@ -63,26 +63,26 @@ class ConversationViewController: MessagesViewController, MessagesDataSource, Me
                    "Pellentesque venenatis, ante et hendrerit rutrum" +
                    "Quam erat vehicula metus, et condimentum ante tellus augue."
 
-        messages.append(MockMessage(text: msg2, sender: sender2, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg4, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg5, sender: sender3, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg1, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg3, sender: sender1, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg3, sender: sender1, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg2, sender: sender2, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg2, sender: sender2, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg1, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg3, sender: sender1, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg2, sender: sender2, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg4, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg5, sender: sender3, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg4, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg5, sender: sender3, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg4, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg5, sender: sender3, id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg1, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg1, sender: currentSender(), id: NSUUID().uuidString))
-        messages.append(MockMessage(text: msg3, sender: sender1, id: NSUUID().uuidString))
+        messages.append(MockMessage(text: msg2, sender: sender2, messageId: NSUUID().uuidString))
```

Comment:
You might have to rebase this / fix the parameter name. The latest `v0.2.0` uses `id`. 
I briefly changed it to `messageId` before disabling the SwiftLint error.